### PR TITLE
update auth command to configure

### PR DESCRIPTION
**Pull request by shereenElSayed**
## Description

Changing `hop auth` to `hop configure` and some of the variables names

## Checklist

* [x] All new functions and classes are documented and adhere to Google doc style (3.8.3-3.8.6 of [this](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods) document)
* [x] Add/update sphinx documentation with any relevant changes.
* [x] Add/update pytest-style tests in `/tests`, ensuring sufficient code coverage.
* [x] `make test` runs without errors.
* [x] `make lint` doesn't give any warnings.
* [x] `make format` doesn't give any code formatting suggestions.
* [x] `make doc` runs without errors and generated docs render correctly.
* [x] Check that CI pipeline run on this PR passes all stages.
* [ ] Review signoff by at least one developer.

NOTE: If this PR relates to a release, open and reference an issue with the Release checklist template.

### CODE REVIEW

**Comment by shereenElSayed** [comment]
@myNameIsPatrick Thanks for your suggestion. I split the classes and did the required changes.

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ can configure a `Stream` instance and pass in an `Auth` instance with credential
 
 ```python
 from hop import Stream
-from hop.auth import Auth
+from hop.configure import Auth
```
Comment (myNameIsPatrick) [inline README.md:102]:
```suggestion
from hop.auth import Auth
```

File: hop/configure.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    else:
+        return os.path.join(os.getenv("HOME"), ".config", auth_filepath)
+
+
+def _add_parser_args(parser):
+    subparser = parser.add_subparsers(title="Commands", metavar="<command>", dest="command")
+    subparser.add_parser(
+        "locate",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        help="display authentication config path",
```
Comment (myNameIsPatrick) [inline hop/configure.py:30]:
```suggestion
        help="display configuration path",
```

File: hop/configure.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    )
+
+    setup_subparser = subparser.add_parser(
+        "setup",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        help="set up authentication config with defaults",
```
Comment (myNameIsPatrick) [inline hop/configure.py:36]:
```suggestion
        help="set up configuration",
```

File: hop/configure.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    )
+    setup_subparser.add_argument(
+        "-f", "--force", action="store_true", help="If set, overrides current configuration",
+    )
+
+
+def _main(args):
+    """Authentication utilities.
```
Comment (myNameIsPatrick) [inline hop/configure.py:44]:
```suggestion
    """Configuration utilities.
```

File: hop/io.py [hunk 5]
```diff
@@ -27,7 +28,7 @@ class Stream(object):
     stream connection is opened, it will use defaults specified here.
 
     Args:
-        auth: A `bool` or `Auth` instance. Defaults to loading from `auth.load_auth()`
```
Comment (myNameIsPatrick) [inline hop/io.py:30]: Most of this should be reverted back, minus the function name `get_config_path()`

Comment (shereenElSayed) [reply hop/io.py]: I changed this comment to "auth" and "Auth" instead of "config" and "Config"

**Pull request merged**